proxy/siip_resolve.py

```diff
@@ -1,58 +1,21 @@
+import sys, os
+sys.path.append(os.path.join(os.path.pardir, 'client'))
+from plow import plow
+
 from siip_certificate import SiipCertificate
 
 def resolve(domain):
-    # TODO dummy data for now. replace with real SIIP resolution
-    domains = {
-        'www.google.com': \
-            SiipCertificate('172.217.4.206', 'www.google.com', \
-                '30:59:30:13:06:07:2A:86:48:CE:3D:02:01:06:08:2A:' + \
-                '86:48:CE:3D:03:01:07:03:42:00:04:59:AD:1B:58:6A:' + \
+    cert_fields = plow(domain)
+    if cert_fields is None:
+        return fallback_resolve(domain)
+    cert_fields = cert_fields.value
+
+    return SiipCertificate(
+        cert_fields.ip,
```

Comment:
I had to change these three lines to:

```python
cert_fields['ip_addr'],
cert_fields['domain'],
cert_fields['key']
```

to get this to work on my machine. Other than that, looks great

Reply:
Good catch. Looks like I really wasn't testing this function at all, because I was just running proxy.py

Reply:
It should be fixed now